runner/debug/debug.go

```diff
@@ -0,0 +1,12 @@
+package debug
+
+import "fmt"
+
+// filterArgs returns a string that contains one '-flagname=filter' pair for each filters element
+func filterArgs(flagname string, filters []string) string {
+	var arguments string
```

Comment:
Since we're building up a string value, I think a `strings.Builder` might be a more intuitive approach than the repeated reassignment of "arguments + something else" to "arguments."

Reply:
I updated this while Dave is away.